website/common/script/content/index.js

```diff
@@ -163,96 +163,101 @@ api.premiumMounts = stable.premiumMounts;
 api.specialMounts = stable.specialMounts;
 api.mountInfo = stable.mountInfo;
 
+// For seasonal events, change these booleans:
+let canUseNormalFood = true;
```

Comment:
so to clarify, I chose canUse as a generic description of both.

If I saw canBuyNormalFood being used in canDrop, I might think it was a mistake.